library/std/src/os/hermit/mod.rs

```diff
@@ -1,14 +1,11 @@
-#![stable(feature = "rust1", since = "1.0.0")]
-#![allow(missing_docs)]
+#![unstable(feature = "hermit_platform", issue = "none")]
```

Comment:
Why did this change?

Reply:
Oh good catch, that is separate change I was working on that should be excluded from this PR. 

I noticed that `hermit` currently exports its extensions as `unix`, even though it is not `target_os = "unix"` and only contains `OsStringExt` (which might be confusing for users expecting e.g. `sys::os::unix::fs::FileExt`). I spoke with someone working on Hermit about whether this was intentional; and he said that currently there is no way for `hermit` to fully replicate everything in `os::unix` and that the plan is to instead move to `os::hermit`.

Reply:
Excluded 👍🏻